src/basics.md

```diff
@@ -1242,6 +1243,49 @@ fn main() {
 }
 ```
 
+[ex-datetime-from-input]: #ex-parse-datetime
+<a name="ex-parse-datetime"></a>
+## Parse string into DateTime struct
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Parses a [`DateTime`] struct from strings representing the well-known formats
+[RFC 2822], [RFC 3339], and a custom format, using
+[`DateTime::parse_from_rfc2822`], [`DateTime::parse_from_rfc3339`], and
+[`DateTime::parse_from_str`] respectively. Escape sequences that are available
+for the [`DateTime::parse_from_str`] can be found at
```

Comment:
We might want to split it into two paragraphs (at least the part about escape sequences)